src/Helper/ControllerHelper.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Helper;
+
+
+class ControllerHelper
+{
+    /**
+     * Calculate the offset for paging
+     *
+     * @param int $page
+     * @param int $linesPerPage
+     * @return float|int
+     */
+    public static function getPagingOffset(int $page, int $linesPerPage)
```

Comment:
Do you really need a Helper for that?

Reply:
It's just the beginning of a Symfony service to help me with paging. It's in progress, I'm gonna improve this.

Reply:
And it's done! I now have a Paginator which saves me from writing duplicated code as in the previous project.